api/base/serializers.py

```diff
@@ -513,8 +513,8 @@ def get_url(self, obj, view_name, request, format):
 
     def to_esi_representation(self, value, envelope='data'):
         relationships = self.to_representation(value)
-        if relationships is not None and 'related' in relationships.keys():
-            href = relationships['related']
+        if relationships is not None and 'links' in relationships.keys() and 'related' in relationships['links'].keys():
```

Comment:
Just verifying that, for our use case, there's nothing alternate paths of this if statement that interest us (such as there is a link in the relationship.keys() but it's not a related link). I'm guessing no, because it's the esi representation and that doesn't care about self links or whatever, but just making sure.